RELEASE/scripts/autoscend/quests/level_04.ash

```diff
@@ -14,28 +14,29 @@ boolean L4_batCave()
 	buffMaintain($effect[Fishy Whiskers], 0, 1, 1);
 
 	int batStatus = internalQuestStatus("questL04Bat");
-	if(batStatus < 3)
+	if (batStatus < 3)
 	{
-		boolean can_use_biscuit = auto_is_valid($item[Sonar-In-A-Biscuit]);
-		if(can_use_biscuit && item_amount($item[Sonar-In-A-Biscuit]) > 0)
+		if (auto_is_valid($item[Sonar-In-A-Biscuit]))
 		{
-			if(use(1, $item[Sonar-In-A-Biscuit]))
+			if (item_amount($item[Sonar-In-A-Biscuit]) == 0 && can_interact())
 			{
-				return true;
+				//if in post ronin or in casual, buy Sonar-In-A-Biscuit if cheaper than what it would cost to get with adventures and clovers.
```

Comment:
sonar in a biscuit has been min priced at 100 meat in the mall since forever.
`use` command in casual causes mafia to mallbuy then use the item.
this is an improvement to the code though since it now properly checks prices and makes sure to buy it separately from using it

Reply:
> sonar in a biscuit has been min priced at 100 meat in the mall since forever.

As last Crimbo proved, things that have been min priced forever can change very abruptly.

Reply:
that is true. thank you for fixing this.